docs/source/kedro_project_setup/settings.md

```diff
@@ -19,3 +21,22 @@ By default, all code in `settings.py` is commented out. When settings are not su
 | `CONFIG_LOADER_CLASS`       | `kedro.config.ConfigLoader`                       | Customise how project configuration is handled.                                                                    |
 | `CONFIG_LOADER_ARGS`        | `dict()`                                          | Keyword arguments for the `CONFIG_LOADER_CLASS` constructor.                                                       |
 | `DATA_CATALOG_CLASS`        | `kedro.io.DataCatalog`                            | Customise how the [Data Catalog](../data/data_catalog.md) is handled.                                              |
+
+## Project metadata
+The `pyproject.toml` file is the standard way to store build metadata and tool settings for Python projects.
+Every Kedro project comes with a default pre-populated `pyproject.toml` file in your project root directory with the following keys specified under the `[tool.kedro]` section:
+
+```toml
+[tool.kedro]
+package_name = package_name
+project_name = project_name
+kedro_init_version = kedro_version
+```
+
+The `package_name` should be a [valid Python package name](https://peps.python.org/pep-0423/) and the `project_name` should be a human-readable name. They are both mandatory keys for your project.
+`kedro_init_version` specifies the version of Kedro the project was created with. When you upgrade to a newer Kedro version,
+this value should also be updated.
```

Comment:
Isn't it the version of the "starter"? Just double checking

Reply:
I think it's the version of Kedro used to create the project. It's also there when you make a blank project with `kedro new`